cli/auth_application.go

```diff
@@ -0,0 +1,96 @@
+package cli
+
+import (
+	"encoding/json"
+
+	"github.com/urfave/cli/v2"
+	apppb "go.viam.com/api/app/v1"
+)
+
+// CreateAuthApplicationAction is the corresponding action for 'third-party-auth-app register'.
+func CreateAuthApplicationAction(c *cli.Context) error {
+	client, err := newViamClient(c)
+	if err != nil {
+		return err
+	}
+
+	return client.createAuthApplicationAction(c)
+}
+
+func (c *viamClient) createAuthApplicationAction(cCtx *cli.Context) error {
+	if err := c.ensureLoggedIn(); err != nil {
+		return err
+	}
+
+	orgID := cCtx.String(generalFlagOrgID)
+	applicationName := cCtx.String(authApplicationFlagName)
+	originURIs := cCtx.StringSlice(authApplicationFlagOriginURIs)
+	redirectURIs := cCtx.StringSlice(authApplicationFlagRedirectURIs)
+	logoutURI := cCtx.String(authApplicationFlagLogoutURI)
+
+	req := &apppb.RegisterAuthApplicationRequest{
```

Comment:
renamed everything to create after convo with natalia. recognize this leaves this gRPC endpoint misnamed... until we have 100% buy in though im not gonna merge the breaking API change.

Reply:
just kidding, renaming back to register